clcache.py

```diff
@@ -177,7 +180,7 @@ def removeObjects(self, stats, removedObjects):
                 rmtree(dirPath)
             stats.setCacheSize(currentSize)
 
-    def getManifestHash(self, compilerBinary, commandLine, sourceFile):
+    def getManifestHash(self, compilerBinary, commandLine, sourceFilePath):
```

Comment:
Why is this argument renaming part of the commit - is `sourceFile` some sort of identifier in Python 3.x?